src/test/java/stormpot/CompoundExpirationTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2011-2014 Chris Vest (mr.chrisvest@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package stormpot;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class CompoundExpirationTest {
+
+    private static final Expiration<Poolable> alwaysExpired = new StaticExpiration(true);
+    private static final Expiration<Poolable> neverExpired = new StaticExpiration(false);
```

Comment:
You can also use `ExpireKit.expire($expired)` and `ExpireKit.expire($fresh)` respectively, instead of creating the `StaticExpiration` class.